app/tests/spec/views/coppa/coppa-age-input.js

```diff
@@ -67,6 +67,24 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('onKeyDown', function () {
+      function testForceNumericInput(element, done) {
+        view.on('preventDefault', function () {
```

Comment:
I think a better way here could be testing for the value of the `#age` field. So your `testForceNumericInput` should do something like: `assert.equal(view.$('#age').val(), '11');`

Example:
https://github.com/mozilla/fxa-content-server/blob/3b40424c1ccbce9dc142f6c0302811c422c697fa/app/tests/spec/views/coppa/coppa-age-input.js#L165